### ISSUE TITLE
Double click to select all text inside brackets in response body pretty view not working

### ISSUE THREAD

**Issue by ankurdengla** [a85/POSTMan-Chrome-Extension#7040]
**Describe the bug**
If the response obtained from a request is of type text/HTML/XML, double-clicking to the immediate right of an opening bracket ( "(" , "[" , or "{" ) should select all the text contained between the opening and respective closing bracket. 

**To Reproduce**
1. Send a request that returns text/html/xml response that has some text between [ ] or ( ) or { }
2. In response, go to Pretty view.
3. Double-click to the immediate right of the bracket.
4. It does not select the whole text contained inside the brackets.

**Expected behavior**
The complete text inside the bracket should be selected.

**Screenshots**
Expected (7.2.2) -
![image](https://user-images.githubusercontent.com/26045713/62292076-25d5e100-b483-11e9-8e7b-3ef22629e2cb.png)

Actual (7.3.4) - 
![image](https://user-images.githubusercontent.com/26045713/62292104-38e8b100-b483-11e9-98c8-123a2d667a8c.png)
 

**App information (please complete the following information):**
 - App Type: Native App
 - Postman Version: 7.3.4 (was working in 7.2.2)
 - OS: macOS Mojave 10.14.3

**Comment by ankurdoshi87** [comment]
The issue still occurs. 
Example if I double click on '{' then the whole inside object get selected in request body. But the same thing doesnt happen in response body.

<img width="881" alt="Screen Shot 2019-08-07 at 6 37 32 PM" src="https://user-images.githubusercontent.com/3383866/62662558-785f4380-b942-11e9-89da-13f81d2a15cf.png">

**Comment by Hayk-S** [comment]
@ankurdoshi87 are you still able to reproduce the issue ?

**Comment by ankurdoshi87** [comment]
Yes. Now its not working for both for Request and Response. It was very useful feature to select the json internal objects.
